stable/insights-agent/requirements.yaml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+  - name: goldilocks
+    repository: https://charts.fairwinds.com/incubator
+    version: 0.2.0
+    condition: goldilocks.enabled
+    values:
```

Comment:
I didn't know you could do this.  Neat.

Reply:
In fact, you cannot. :(

Moved to values.yaml

Reply:
😆